policyengine_us/parameters/gov/states/in/tax/income/agi_rate.yaml

```diff
@@ -1,13 +1,11 @@
-description: IN adjusted gross income tax rate.
-values:
-  2014-01-01: 0.034 # "For taxable years beginning before January 1, 2015"
-  2015-01-01: 0.033
-  2017-01-01: 0.0323
+description: Indiana adjusted gross income tax rate.
 metadata:
   period: year
   unit: /1
-  name: in_agi_tax_rate
-  label: IN AGI tax rate
   reference:
```

Comment:
```suggestion
  label: Indiana AGI tax rate
```

Reply:
Added label: to agi_rate.yaml parameter file.